client/blocks/product-selector/index.jsx

```diff
@@ -164,9 +164,9 @@ export class ProductSelector extends Component {
 			return;
 		}
 
-		return translate( 'Purchased %(purchaseDate)s', {
+		return translate( 'Purchased on %(purchaseDate)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.